build.gradle

```diff
@@ -39,49 +36,22 @@ java {
 }
 
 publishing {
-    publications {
-        maven(MavenPublication) {
+	publications {
+		maven(MavenPublication) {
 			from components.java
-        }
-    }
-    repositories {
-		repositories {
-			if (ENV.MAVEN_URL) {
-				maven {
-					url ENV.MAVEN_URL
-					credentials {
-						username ENV.MAVEN_USERNAME
-						password ENV.MAVEN_PASSWORD
-					}
+		}
+	}
+	repositories {
+		if (System.getenv('MAVEN_URL')) {
+			maven {
+				url System.getenv('MAVEN_URL')
+				credentials {
+					username System.getenv('MAVEN_USERNAME')
+					password System.getenv('MAVEN_PASSWORD')
 				}
 			}
 		}
-    }
-}
-
-curseforge {
```

Comment:
Ill add this back.

Reply:
oh i didnt know fabric was on curseforge anymore

Reply:
Its fine, I would have also removed this plugin as I have my own. Ill sort this out after merging.